### ISSUE TITLE
TP: Remove irrelevant context menu items from any map ds

### ISSUE THREAD

**Issue by mitchell852** [apache/trafficcontrol#4231]
## I'm submitting a ...

- [x] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [ ] Traffic Ops
- [ ] Traffic Ops ORT
- [x] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:
![image](https://user-images.githubusercontent.com/251272/71380854-1c622800-258e-11ea-94d5-01b01c409a94.png)

The following items do not pertain to any map ds's and should be removed from the context menu for any map ds's:

Manage Invalidation Requests
Manage Origins
Manage Regexes
Manage Static DNS Entries

## Expected / new behavior:

Manage Invalidation Requests
Manage Origins
Manage Regexes
Manage Static DNS Entries

are all removed from the any map ds context menu.

## Minimal reproduction of the problem with instructions:
Navigate to https://tp.domain.net/#!/delivery-services and right-click or control-click on an any map delivery service to show the context menu. Notice the menu items that do not apply to any map ds's.

**Comment by ocket8888** [comment]
I think you can still have content invalidation jobs for ANY_MAP DSes, also unsure about Static DNS Entries

**Comment by mitchell852** [comment]
> I think you can still have content invalidation jobs for ANY_MAP DSes

No, you can't. If you try to create an invalidation job on a DS that has no origin (i.e. any map ds's), the job is rejected.

Also, Static DNS entries is used by Traffic Router and any map's are unknown to TR.
